src/main/java/solutions/fairdata/openrefine/metadata/fdp/VocabularyHelper.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * The MIT License
+ * Copyright © 2019 FAIR Data Team
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package solutions.fairdata.openrefine.metadata.fdp;
+
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import org.eclipse.rdf4j.model.vocabulary.DCAT;
+import org.eclipse.rdf4j.model.vocabulary.RDF;
+
+public class VocabularyHelper {
+
+    private static final SimpleValueFactory vf = SimpleValueFactory.getInstance();
+    // "Shared"
+    public static final IRI TYPE = RDF.TYPE;
+    public static final IRI TYPE_CATALOG = DCAT.CATALOG;
+    public static final IRI TYPE_DATASET = DCAT.DATASET;
+    public static final IRI TYPE_DISTRIBUTION = DCAT.DISTRIBUTION;
+    public static final IRI TITLE = vf.createIRI("http://purl.org/dc/terms/title");
+    public static final IRI VERSION = vf.createIRI("http://purl.org/dc/terms/hasVersion");
+    public static final IRI DESCRIPTION = vf.createIRI("http://purl.org/dc/terms/description");
+    public static final IRI LICENSE = vf.createIRI("http://purl.org/dc/terms/license");
+    public static final IRI RIGHTS = vf.createIRI("http://purl.org/dc/terms/accessRights");
+    public static final IRI PUBLISHER = vf.createIRI("http://purl.org/dc/terms/publisher");
+    public static final IRI PUBLISHER_NAME = vf.createIRI("http://xmlns.com/foaf/0.1/name");
+    public static final IRI PARENT = vf.createIRI("http://purl.org/dc/terms/isPartOf");
+    public static final IRI LANGUAGE = vf.createIRI("http://purl.org/dc/terms/language");
```

Comment:
Do you need the custom constant names, or could the DC Terms constants from rdf4j be reused?
```java
import org.eclipse.rdf4j.model.vocabulary.DCTERMS;
public static final IRI TITLE = DCTERMS.TITLE;
```

Reply:
Fixed in https://github.com/FAIRDataTeam/OpenRefine-metadata-extension/pull/43/commits/85febfd684c2a63f144bf246e26e5703548e0897 and it passed E2E tests, so am I going to merge and release. Thanks for the review 👍